File: Iniciante/1164.py
def ehPfto(n):
  soma = 0
  for i in range(1, n):
    if (n%i == 0):
      soma += i

  return soma == n

n = int(input())

for i in range(n):
  x = int(input())
  
  if (ehPfto(x)):
    print("{} eh perfeito".format(x))
  else:
    print("{} nao eh perfeito".format(x))

File: Iniciante/1164.js
function ehPfto(n) {
  let soma = 0;
  for (let i = 1; i < n; i++) {
    if (n % i === 0) {
      soma += i;
    }
  }
  return soma === n;
}

let n = Number(lines.shift());

for (let i = 0; i < n; i++) {
  let x = Number(lines.shift());
  
  if (ehPfto(x)) {
    console.log(`${x} eh perfeito`);
  } else {
    console.log(`${x} nao eh perfeito`);
  }
}
